k2/csrc/fsa.h

```diff
@@ -12,6 +12,10 @@
 
 namespace k2 {
 
+using Label = int32_t;
+using StateId = int32_t;
+using Weight = float;
```

Comment:
let's just keep this as float for now.  Eventually things may be templated so we have float and double versions, for integration with PyTorch/TF.

Reply:
Sure